frontend/i18n-scripts/memsource-upload.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -exuo pipefail
+
+# Before running this tool, you must first install the unofficial Memsource CLI client: https://github.com/unofficial-memsource/memsource-cli-client#pip-install
+# You also have to configure it with your login info: https://github.com/unofficial-memsource/memsource-cli-client#configuration-red-hat-enterprise-linux-derivatives
+
+# Example CLI usage: yarn memsource-upload -v 4.8 -n "UI Localization - Sprint 200"
+
+while getopts v:n: flag
+do
+    case "${flag}" in
+        v) VERSION=${OPTARG};;
+        n) NAME=${OPTARG};;
+    esac
+done
+
+echo "Creating project with title \"[OCP $VERSION] $NAME\""
+
+PROJECT_INFO=$(memsource project create --name "[OCP $VERSION] $NAME" --template-id 169304 -f json)
+PROJECT_ID=$(echo "$PROJECT_INFO" | jq -r '.uid')
+
+echo "Exporting PO files"
+yarn export-pos
+
+echo "Exported all PO files"
+
+jaFiles=( po-files/ja/*.po )
```

Comment:
We might consider adding a list of languages somewhere and making this a loop. That way adding or removing languages in the future is easier, and we're not duplicating commands.